static/app/views/settings/organizationTeams/teamMembersRow.tsx

```diff
@@ -0,0 +1,150 @@
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import IdBadge from 'sentry/components/idBadge';
+import {PanelItem} from 'sentry/components/panels';
+import RoleSelectControl from 'sentry/components/roleSelectControl';
+import {IconSubtract} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Member, Organization, User} from 'sentry/types';
+import {
+  hasOrgRoleOverwrite,
+  RoleOverwriteIcon,
+} from 'sentry/views/settings/organizationTeams/roleOverwriteWarning';
+
+type Props = {
```

Comment:
You could specify this to be `rowProps` instead so you don't need to worry about conflicting with the select props below